src/overlays/effects/ovl_Effect_Ss_Extra/z_eff_ss_extra.

```diff
@@ -0,0 +1 @@
+
```

Comment:
Seems like you accidentally created an extra empty file

Reply:
well that sure is odd... hmm